Dockerfile.dapper

```diff
@@ -1,13 +1,22 @@
-FROM registry.suse.com/bci/bci-base:15.3
+FROM ghcr.io/oracle/oraclelinux:7-slim
 
 ARG DAPPER_HOST_ARCH
 ENV ARCH $DAPPER_HOST_ARCH
 
-RUN zypper -n in git-core curl tar gzip docker wget awk hostname xz && \
-    zypper -n clean -a && \
-    rm -rf /tmp/* /var/tmp/* /usr/share/doc/packages/*
+RUN yum-config-manager --enable ol7_optional_latest && \
+    yum-config-manager --enable ol7_addons
 
-RUN curl -sL https://github.com/koalaman/shellcheck/releases/download/stable/shellcheck-stable.linux.x86_64.tar.xz | tar xvJf - -C /usr/local/bin --strip-components=1
+RUN yum update -y && \
+    # software collections repo needed for git 2.x on OL7
+    yum-config-manager --add-repo=http://yum.oracle.com/repo/OracleLinux/OL7/SoftwareCollections/x86_64 && \
+    yum install -y bash rh-git227 gcc glibc-static docker-cli vim awk less libffi file curl wget ca-certificates unzip tar jq && \
+    # Set up needed to ensure git 2.27 from rh-git227 is on the path
```

Comment:
You should do this to remove unneeded yum files
    && yum clean all \
    && rm -rf /var/cache/yum

Reply:
fixed